.github/workflows/verifyVersion.js

```diff
@@ -0,0 +1,102 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @format
+ */
+
+module.exports = async (github, context) => {
+  const issue = context.payload.issue;
+
+  // Ignore issues using upgrade template (they use a special label)
+  if (issue.labels.find(label => label.name == 'Type: Upgrade Issue')) {
+    return;
+  }
+
+  // Extract RN version number from the issue body, if it exists
+  const getReactNativeVersionIfExists = issue => {
+    if (!issue || !issue.body) return;
+    const rnVersionRegex =
+      /React Native Version[\r\n]+[0-9]{1,2}\.[0-9]{1,2}\.[0-9]{1,2}/;
+    const rnVersionMatch = issue.body.match(rnVersionRegex);
+    if (!rnVersionMatch || !rnVersionMatch[0]) return;
+    return rnVersionMatch[0];
+  };
+  const issueVersion = parseVersionFromString(
+    getReactNativeVersionIfExists(issue),
+  );
+
+  if (!issueVersion) {
+    return 'Needs: Version Info';
+  }
+
+  // Ensure the version matches one we support
+  const recentReleases = (
+    await github.rest.repos.listReleases({
+      owner: context.repo.owner,
+      repo: context.repo.repo,
+    })
+  ).data.map(release => release.name);
+
+  const latestRelease = (
+    await github.rest.repos.getLatestRelease({
+      owner: context.repo.owner,
+      repo: context.repo.repo,
+    })
+  ).data;
+  const latestVersion = parseVersionFromString(latestRelease.name);
+
+  if (!isVersionSupported(issueVersion, latestVersion, recentReleases)) {
+    return 'Needs: Verify on Latest Version';
+  }
+};
+
+// We support N-2 minor versions. We also want to match the highest available patch for the given minor.
+// E.g. the latest release is 0.71.2. 0.71.0 -> Please upgrade. 0.69.8 -> OK. 0.69.7 -> Please upgrade.
```

Comment:
I agree with some other folks concerns that if we try to push people to always be on latest patch available for a specific minor, we might create some friction and potentially some noise.

But, honestly, it's... probably worth it. If it gets too much then we can revisit

Reply:
We can use this as an opportunity to create a distinction between the two labels we have now that are similar but slightly different: 
1. "Needs: Verify on Latest Version" and
2. "Type: Unsupported Version"

We can use 1. for supported minor versions that aren't using the latest patch. I can even update the script to suggest the user upgrade to the highest available patch for the minor, rather than "latest" in every case.
And we can use 2. for truly unsupported versions, anything that is older than N-2 from the current minor.

As a further measure, if we'd like to soften the messaging, we can also not replace the Needed: Triage tag with Needed: Author Feedback tag in the first case, if we want to treat it as a suggestion.

I am definitely also open to iterating on this if the first solution we use ends up causing too much noise or churn for people.

Reply:
I think your idea is great, let's go for this adjustment 👍